content/en/docs/concepts/architecture/unknown-version-interoperability-proxy.md

```diff
@@ -0,0 +1,79 @@
+loads a special filter that identifies API groups / versions / resources that the local server doesn't recognise,
+and tries to proxy those requests to a peer API server that is capable of handling the request.
+
+When a kube-apiserver receives a resource request, it first checks which API servers can serve the
+requested resource. The check happens using the internal `StorageVersion` API.
+If the resource is known to the local API server, the request is handled locally. If there is no 
+internal `StorageVersion` object found for the requested resource and the configured APIService
+specifies proxying to an extension API server, that proxying happens following the usual
+[flow](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-aggregation-layer/) for
+extension APIs.
+
+Once a valid internal `StorageVersion` object is found for the requested resource, 
+the API server trying to handle the request (the _handling API server_) fetches the 
+peer API servers that do serve the relevant API group / version / resources using the information 
+in the fetched `StorageVersion` object. The _handling API server_ then proxies the request
+to one of the matching peer kube-apiservers that are aware of the requested resource.
+If there is no peer known for that API group / version / resource, the handling API server passes the 
+request to its own handler chain which should eventually return a 404("Not Found") response.
+If the handling API server has identified and selected a peer API server, but that peer fails to respond
+(for reasones such as network connectivity, or a data race between the request being received and
+a controller registering the peer's info into an internal `ServerStorageVersion` object), then the
+handling API server responds with a 503 (“Service Unavailable”) error.
+
+## Proxy transport and authentication between API servers {#transport-and-authn}
+
+To authenticate the destination server's serving certs, you must configure a certificate authority
+bundle by specifying the `--peer-ca-file` command line argument to the **source** API server.
+
+The source kube-apiserver reuses the existing `--proxy-client-cert-file` and `--proxy-client-key-file` to
+present its identity that will be verified by its peer (the destination kube-apiserver). The destination API
+server verifies that peer connection based on the configuration you specify using the
+`--requestheader-client-ca-file` command line argument.
+
+## Configuration for peer API server connectivity
+
+To find the network location of the peer kube-apiserver, you use the 
+`--peer-advertise-ip` and `--peer-advertise-port` command line arguments to kube-apiserver or specify 
+these fields in the API server configuration file.
+If these flags are unspecified, the kube-apiserver uses the value from either `--advertise-address` or
+`--bind-address` flag. If those too, are unset, the host's default interface is used.
```

Comment:
Combine with the above `Enabling the Unknown Version Interoperability Proxy` section so that all the setup/configuration is explained together?

Reply:
Done